backend/db.js

```diff
@@ -53,13 +59,31 @@ export const getLineOnPage = async (
 ) => getLinesOnPageBase( sourceId, page )
   .offset( lineIndex )
   .first()
-  .then( ( { id } ) => Lines.query().where( 'id', id ).withTranslations() )
+  .then( ( { id } ) => Lines.query().where( 'id', id ).withTranslations( query => query.joinEager( 'translationSource' ) ) )
   .then( ( [ { translations, ...line } ] ) => ( {
```

Comment:
Need `async`

Reply:
Where/Why?

Reply:
In line 65 you use await but there is no async at line 63. When I ran it linter picked it up.